packages/chaire-lib-backend/src/config/server.config.ts

```diff
@@ -191,6 +353,35 @@ export const setProjectConfiguration = (newConfig: Partial<ProjectConfiguration<
             }
         });
     }
+
+    // If osrmRouting is configured using the old defaultPreferences format and not the new one in server.config, set it to the new format
+    // FIXME Remove once the old osrmRouting format is fully deprecated
```

Comment:
Could be nice to have an issue number here, to make it easier to find the code to remove.